1-development/client-from-the-bahamas.md

```diff
@@ -0,0 +1,28 @@
+InvoiceXpress is a web app that makes online invoicing easy. Customers can create an account and use the API to create invoices. Each invoice has a `Client`, and there is an [API for interacting with clients](http://invoicexpress.com/api/clients/create).
+
+### Bahamas
+
+It has come to our attention that on the Bahamas, invoices can have an additional client, that will be eligible for tax deductions. We need a service to handle that specific logic. The service would be called by the main app to store and retrieve the `Client`.
```

Comment:
This was the phrase that got me confused. I think it should say: "The service would be called by **our main app** to store and retrieve the `Client`". Just change the 'the' to 'our'.